src/schema/landingPage/resolvers.ts

```diff
@@ -7,27 +7,69 @@ const normalizeLandingPage = (collection) => {
   let normalizedLandingPage = normalizeKeys(collection);
   const normalizedKeys = [
     'title',
+
+    // BEGIN deprecated fields
     'description',
+    'socialMediaImage',
     'titleAndDescriptionColor',
     'buttonText',
     'buttonUrl',
     'heroBackgroundImage',
     'heroBackgroundImageMobile',
     'heroBackgroundImageColor',
     'heroTopLayerImage',
+    // END deprecated fields
+
     'keywords',
-    'socialMediaImage',
     'metaInformation',
     'pageTitle',
+    {
+      bottomBanner: [
+        'title',
+        'description',
+        'titleAndDescriptionColor',
+        'buttonText',
+        'buttonUrl',
+        'heroBackgroundImage',
+        'heroBackgroundImageMobile',
+        'heroBackgroundImageColor',
+        'heroTopLayerImage',
+        'socialMediaImage',
+      ],
+    },
+    {
+      topBanner: [
+        'title',
+        'description',
+        'titleAndDescriptionColor',
+        'buttonText',
+        'buttonUrl',
+        'heroBackgroundImage',
+        'heroBackgroundImageMobile',
+        'heroBackgroundImageColor',
+        'heroTopLayerImage',
+        'socialMediaImage',
+      ],
+    },
```

Comment:
duplicated keys here, could maybe be in a array variable

Reply:
and would helper function that recursively goes through all the field make sense for normalize functions?

Reply:
Is there a risk that same field name in some other place is not localizedObject? If not, then we could just have flat array of field names and recursively modify the object

Reply:
This is actually something that could be done on CMS side as well. So instead of returning e.g. 
```
"hero_background_image_mobile_fi": null,
"hero_background_image_mobile_sv": null,
"hero_background_image_mobile_en": null,
```
CMS could return
```
"hero_background_image_mobile {
  "en": null,
  "fi": null,
  "sv": null
}
```
So basically same thing what e.g. Linked Events does

Reply:
That's actually a good idea, I was the one who implemented the API without the prior knowledge of how it works in GraphQL proxy. Since I haven't merged that change yet, I can change it from CMS side, let see how it goes. I'll update the PR soon